src/main/java/Duke.java

```diff
@@ -1,10 +1,112 @@
+import java.util.ArrayList;
```

Comment:
overall the code adheres to the coding guidelines, but there are some minor changes that can be made to improve on code readability, as suggested by the other reviewer.